github-actions/docs/do-we-build-and-if-so-what.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * DECLARATIONS
+ */
+$event = getenv('GITHUB_EVENT_NAME') ?: '';
```

Comment:
Verified locally that env passing works by doing the following:

- Created a `test.php` script:
  ```php
  <?php
  $event = getenv('GITHUB_EVENT_NAME');
  echo "RECEIVED: " . $event;
  ```

- Created a `test.sh` script:
  ```bash
  #!/bin/bash
  RESULT=$(php test.php)
  echo "RESULT:"
  echo ${RESULT}
  ```

And then executing as follows:

```bash
$ export GITHUB_EVENT_NAME=something
$ ./test.sh
```

Which resulted in:

```text
RESULT:
RECEIVED: something
```